samples/matter/window_covering/sample.yaml

```diff
@@ -0,0 +1,42 @@
+sample:
+  description: Matter Window Covering example
+  name: Matter Window Covering
+tests:
+  sample.matter.window_cover.debug:
+    build_only: true
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+  sample.matter.window_cover.low_power:
+    build_only: true
+    extra_args: OVERLAY_CONFIG=overlay-low_power.conf
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+  sample.matter.window_cover.no_dfu:
+    build_only: true
+    extra_args: CONF_FILE=prj_no_dfu.conf
+    integration_platforms:
+      - nrf21540dk_nrf52840
+    platform_allow: nrf21540dk_nrf52840
```

Comment:
I think nrf52 and nrf53 are also supported, so could be added here (but good that they're not included in integation_platforms).